kitchen.macos.yml

```diff
@@ -0,0 +1,15 @@
+---
+driver:
+  name: exec
+  host: localhost
+
+provisioner:
+  require_chef_omnibus: false
+  name: chef_zero
+  install_strategy: skip
+  chef_client_path: "/opt/chef-workstation/bin/chef-client"
+  deprecations_as_errors: false
+  sudo: true
+
+platforms:
+  - name: macos
```

Comment:
You might look at what we did in the ntp cookbook for MacOS testing using the exec provider. I think it might be a better way to go.